README.md

```diff
@@ -27,17 +30,20 @@ If you don't want to use Docker, you need:
 To setup your Docker instance simply follow these steps:
 1. Pull the `ghcr.io/vatsim-scandinavia/control-center:v4` Docker image
 2. Setup your MySQL database (not included in Docker image)
-3. Configure the environment variables as described in the [CONFIGURE.md](CONFIGURE.md)
-4. Run the container
+3. Configure the environment variables as described in the [CONFIGURE.md](CONFIGURE.md#environment)
+4. Run the container and access it with `docker exec -it control-center bash`
+    - Run `php artisan migrate` to setup the database
+    - To ensure that users will not need to login after each time you re-deploy/upgrade the container, run `php artisan key:get`. Copy the key and set it as the `APP_KEY` environment variable in your docker configuration.
```

Comment:
````suggestion
4. Start the container in the background.
5. Setup the database.
   ```sh
   docker exec -t control-center php artisan migrate
   ```
6. To ensure that users will not need to log in after each time you re-deploy or upgrade the container, you need to create and store an application key in your environment.
   ```sh
   docker exec -t control-center php artisan key:get
   ```
   Copy the key and set it as the `APP_KEY` environment variable in your Docker configuration.
````

WDYT?

Reply:
This will work for the key, but not artisan migrate, because migrate will ask you "are you sure, bc its prod" and docker exec doesn't support this and ends the execute after it's called and gets first reply. This gets more tricky with `php artisan user:makeadmin` that will ask multiple questions etc. Migrate prolly has some force argument, but makeadmin doesn't (yet).

I think we should look into this in a separate PR, because I agree docker exec would be even easier to use for most users. Though I don't want to mess with this 5 min before pushing this

Reply:
That's a good point, but `docker exec` supports it: just needs `-it` instead of `-t`, no?

Reply:
Good point, added in commit